dcs_deploy.py

```diff
@@ -446,31 +703,131 @@ def load_selected_config(self):
-            external_xml_config_path = os.path.join(self.l4t_root_dir, 'tools/kernel_flash/flash_l4t_external_custom.xml')
-            os.chdir(self.l4t_root_dir)
+    def setup_initrd_flashing(self):
+        os.chdir(self.l4t_root_dir)
+        #set variables for initrd flash
+        self.flash_script_path = os.path.relpath('tools/kernel_flash/l4t_initrd_flash.sh')
+        
+        if self.config['device'] == 'xavier_nx':
+            self.board_name = 'airvolute-dcs' + self.config['board'] + "+p3668-0001-qspi-emmc"
+            self.orin_options = ""
+        elif self.config['device'] == 'orin_nx':
+            self.board_name = 'airvolute-dcs' + self.config['board'] + "+p3767-0000"
+            # based on docu from tools/kerenel_flash/README_initrd_flash.txt and note for Orin (Workflow 4)
+            # sudo ./tools/kernel_flash/l4t_initrd_flash.sh --external-device nvme0n1p1 -c tools/kernel_flash/flash_l4t_external.xml -p "-c bootloader/t186ref/cfg/flash_t234_qspi.xml --no-systemimg" --network usb0      <board> external
+            self.orin_options = '--network usb0 -p "-c bootloader/t186ref/cfg/flash_t234_qspi.xml --no-systemimg"'
+        elif self.config['device'] == 'orin_nx_8gb':
+            self.board_name = 'airvolute-dcs' + self.config['board'] + "+p3767-0001"
+        elif self.config['device'] == 'orin_nano_8gb':
+            self.board_name = 'airvolute-dcs' + self.config['board'] + "+p3767-0003"
+            self.orin_options = '--network usb0 -p "-c bootloader/t186ref/cfg/flash_t234_qspi.xml --no-systemimg"'
+        elif self.config['device'] == 'orin_nano_4gb':
+            self.board_name = 'airvolute-dcs' + self.config['board'] + "+p3767-0004"
+            self.orin_options = '--network usb0 -p "-c bootloader/t186ref/cfg/flash_t234_qspi.xml --no-systemimg"'
+        else:
+            print("Unknown device! [%s] exitting" % self.config['device'])
+            exit(8)
+         
+        if self.config['storage'] == 'emmc':
+            self.rootdev = "mmcblk0p1"
+            self.external_device = ""
+        elif self.config['storage'] == 'nvme':
+            self.rootdev = "external"
+            self.external_device = "--external-device nvme0n1p1 "
+            if self.args.ab_partition == True:
+                # setup multiple app partitions
+                self.ext_partition_layout = os.path.relpath('tools/kernel_flash/flash_l4t_nvme_rootfs_ab.xml')
+            else:
+                # setup no multiple app partitions
+                self.ext_partition_layout = os.path.relpath('tools/kernel_flash/flash_l4t_external_custom.xml')
+        else:
+            print("Unknown storage [%s]! exitting" % self.config['storage'])
+            exit(9)
+        # fix default rootdev to external  (or internal) for orin. There is NFS used to flash
+        if self.config['device'] == 'orin_nx' or self.config['device'] == 'orin_nx_8gb' or self.config['device'] == 'orin_nano_8gb' or self.config['device'] == 'orin_nano_4gb':
```

Comment:
More compact and more readable version
```suggestion
        if self.config['device'] in ['orin_nx', 'orin_nx_8gb', 'orin_nano_8gb', 'orin_nano_4gb']:
```